amr-wind/wind_energy/ABLFieldInit.cpp

```diff
@@ -158,13 +194,15 @@ void ABLFieldInit::operator()(
     const amrex::Real rho_init = m_rho;
 
     const int ntvals = static_cast<int>(m_theta_heights.size());
+    const int nwvals = static_cast<int>(m_wind_heights.size());
     const amrex::Real* th = m_thht_d.data();
     const amrex::Real* tv = m_thvv_d.data();
 
-    if (m_init_uvtheta_profile) {
+    if (m_init_uvtheta_profile || m_initial_wind_profile) {
```

Comment:
Is there always going to be a initial wind profile when there is a uvtheta profile? I'm wondering how this change won't break the workflow/crash when there is just one and not the other.

Reply:
uvtheta profile is something the code already had and I wanted to add the wind profile as an extra parameter on top of it.

Reply:
Doesn't this change break the workflow for cases without a wind profile (but have a uvtheta profile)? i.e. What's preventing a segfault on `windh[iz]` in that setup? You've not just added this atop, but removed the existing behavior for solpeu/slopev calcs right? 

https://github.com/Exawind/amr-wind/blob/b021068625a0a75af94aeedc9e786be752866511/amr-wind/wind_energy/ABLFieldInit.cpp#L201-L235

Reply:
I see what you mean. I had an if check inside which had vanished. Will update this.